doc/unison-manual.tex

```diff
@@ -132,31 +132,37 @@
 \URL{http://research.microsoft.com/\home{fournet}/}{Cedric Fournet}
 contributed to an even earlier prototype.
 
-\TOPSUBSECTION{Mailing Lists and Bug Reporting}{lists}
```

Comment:
Removing this sections breaks a cross-ref in section "Going Further".

Reply:
fixed I think